lerna.json

```diff
@@ -0,0 +1,15 @@
+{
+    "packages": [
+        "packages/*"
+    ],
+    "version": "5.0.0",
+    "command": {
+        "bootstrap": {
+            "hoist": true
+        },
+        "publish": {
```

Comment:
Shouldn't this be `version` instead of `publish`? I can't find any mention to `conventionalCommits` in the [docs](https://github.com/lerna/lerna/tree/master/commands/publish#readme).